openedx/core/lib/edx_api_utils.py

```diff
@@ -15,7 +15,7 @@
 
 
 def get_edx_api_data(api_config, user, resource,
-                     api=None, resource_id=None, querystring=None, cache_key=None):
+                     api=None, resource_id=None, querystring=None, cache_key=None, collection=True):
```

Comment:
@mattdrayer, what about flipping it? The current API assumption is that we're retrieving a... plurality... of items, so what if instead of defaulting to `multiple=True`, we defaulted to `single_item=False`?